compiler/llstream.nim

```diff
@@ -70,10 +70,10 @@ proc llStreamClose*(s: PLLStream) =
 when not declared(readLineFromStdin):
   # fallback implementation:
   proc readLineFromStdin(prompt: string, line: var string): bool =
-    stdout.write(prompt)
+    stderr.write(prompt)
```

Comment:
Huh? A prompt to stderr? Why?

Reply:
Well, the compiler writes errors to stderr, and sometimes the prompt is printed before the error, for example:
`>>> stdin(3, 6) Error: undeclared identifier: 'y'`
instead of:
`stdin(3, 6) Error: undeclared identifier: 'y'`
`>>> `

As you said stdout/stderr is a bug :) So I think using stderr in this case for uniformity with the compiler output is not such a bad idea.

Reply:
ok